docs/tables/yml_key_value.md

```diff
@@ -0,0 +1,242 @@
+# Table: yml_key_value
+
+Query key-value pair data along with comments and line number from YML files found in the configured `paths`.
+
+For instance, if `paths` is set to `/Users/myuser/yml/*`, and that directory contains:
+
+- sample.yml
+- invoice.yml
+
+This table will retrieve all key-value pairs from each file mentioned above, along with comments and the line number, which you can then query directly:
+
+```sql
+select
+  key_path,
+  value,
+  tag as value_type,
+  pre_comments,
+  start_line
+from
+  yml_key_value;
+```
+
+```sh
++----------------------+---------------------------------------------------------------------------------------------------+------------+-----------------------------+------------+
+| key_path             | value                                                                                             | value_type | pre_comments                | start_line |
++----------------------+---------------------------------------------------------------------------------------------------+------------+-----------------------------+------------+
+| items.1.size         | 8                                                                                                 | integer    | []                          | 15         |
+| specialDelivery      | Follow the Yellow Brick Road to the Emerald City. Pay no attention to the man behind the curtain. | string     | []                          | 25         |
```

Comment:
This line ends up stretching our Hub page, can you please remove the `specialDelivery` key here, in other examples below, and in the `json_key_value` doc (for consistency)?